### PR TITLE
Add nacos discovery native-image AOT configuration files

### PR DESCRIPTION
**Pull request by ruansheng8**

### CODE REVIEW

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/jni-config.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/jni-config.json]: Don't include empty file, same problem need to be done below.

Comment (ruansheng8) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/jni-config.json]: Done

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/serialization-config.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/serialization-config.json]: Delete files that have no actual content.

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/predefined-classes-config.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/predefined-classes-config.json]: Delete files that have no actual content.

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/serialization-config.json (hunk 2 above)
Comment (ruansheng8) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/serialization-config.json]: Done

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/predefined-classes-config.json (hunk 3 above)
Comment (ruansheng8) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/predefined-classes-config.json]: Done

**Review by steverao — APPROVED** [review]

**Pull request merged**